src/components/30-organisms/modal/story.js

```diff
@@ -18,10 +18,23 @@ export default {
 
 export const Modal = () => {
   const open = boolean('open', true);
+  const forced = boolean('forced', false);
 
   const wrapper = document.createElement('div');
+
+  setTimeout(() =>
+    document.querySelector('axa-button').addEventListener('click', () => {
+      document.querySelector('axa-modal').removeAttribute('open');
+    })
+  );
+
```

Comment:
Do you need `setTimeout()` is there no other way like we did on line 64?

Reply:
No that's not possible without using a native button (instead of our webcomponent).